task/09-functions-n-closures-tasks.js

```diff
@@ -104,7 +111,19 @@ function memoize(func) {
  * retryer() => 2
  */
 function retry(func, attempts) {
-    throw new Error('Not implemented');
+    return () => {
+
+        while(true){
```

Comment:
it's a bad practice to use while(true) when loop doesn't have to be infinite. Add exit condition to the while statement. Or you since its a loop with fixed max number of iteration you could use for-loop.